app/views/layouts/application.html.slim

```diff
@@ -42,7 +42,7 @@ html lang="#{I18n.locale}"
     .site-wrap
       = render 'shared/i18n_mode' if FeatureManagement.enable_i18n_mode?
       = render 'shared/usa_banner'
-      - if signed_in?
+      - if user_fully_authenticated?
```

Comment:
I agree. Would it make sense to break this up into multiple smaller PRs that tackle one feature at a time? I see the following changes that perhaps are not dependent on each other?

- Cancel link
- Refactoring instructions and fallback text to be consistent
- Removing the resend flash
- Displaying the nav/auth only to fully signed in users